crates/ty_python_semantic/resources/mdtest/type_properties/is_equivalent_to.md

```diff
@@ -607,23 +607,33 @@ module:
 `module2.py`:
 
 ```py
-import importlib
-import importlib.abc
+import imported
+import imported.abc
+```
+
+`imported/__init__.pyi`:
```

Comment:
I think it's fine (even good) to make this change, but I assume the previous version of this test (using `importlib`) still passes after this PR? It's just that the simpler version was easier to debug in development?

Reply:
Different versions of this PR have variously broken and unbroken the test because of the line `from importlib.abc import Loader` in typeshed's `importlib/__init__.pyi`. I would rather check in this not-relying-on-a-random-library version.